tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -175,107 +191,121 @@ Feature: an user gets the resources shared to them
                       "textfile0.txt"
                     ]
                   },
+                  "parentReference": {
+                    "type": "object",
+                    "required": [
+                      "driveId",
+                      "driveType"
+                    ],
```

Comment:
for `parentReference` do we get like this in response https://github.com/owncloud/ocis/blob/4d5490a60135df8beb8a64626192e6555858f74e/tests/acceptance/features/apiSharingNg/sharedByMe.feature#L37-L45. may be we could also extend to it if we get them in the reponse.

Reply:
we only get these props
```json
"driveId",
"driveType"
```